spec/controllers/dashboard_controller_spec.rb

```diff
@@ -115,7 +115,7 @@
       auth_token     = "aabbccddeeff"
       validation_url = "/user_validation_url"
 
-      request.env["HTTP_X_REMOTE_USER"] = user.userid
+      request.env["HTTP_X_REMOTE_USER"] = "JohnDoe"
```

Comment:
minor, maybe leverage what's already there ?   user.userid.upcase or user.userid.capitalize ?

Reply:
👍 Good idea! Thank you @abellotti